README.md

```diff
@@ -42,7 +42,7 @@ Forking the project will create a copy of that project in your own GitHub accoun
 
 
 #### Updating your local
-In order to update your local environment to the latest version on `development`, you will have to pull the changes using the `upstream` repository, for example: `git pull upstream development`. This will pull all the new commits from the origin repository to your local environment.
+In order to update your local environment to the latest version on `development`, you will have to pull the changes using the `upstream` repository, for example: `git pull upstream development` or just `git pull`. This will pull all the new commits from the origin repository to your local environment.
```

Comment:
Please remove "or just git pull", I believe this is the problem you can't load scss files xD You are not pulling from upstream to get the latest changes.